jobs/openshift/cluster/integreatly/test/Jenkinsfile

```diff
@@ -68,12 +81,58 @@ pipeline {
           ]
           build job: jobName, parameters: jobParams
         }
-      }
+      } 
     }
 
     stage('Test Integreatly') {
       steps {
-        echo "[INFO] Running Tests on Integreatly"
+        echo "[INFO] Running Browser Tests on Integreatly"
+        script { 
+          jobName = 'browser-based-testsuite-pipeline'
+          jobParams = [
+            [$class: 'StringParameterValue', name: 'REPOSITORY', value: QE_REPO_URL],
+            [$class: 'StringParameterValue', name: 'BRANCH', value: 'master'],
```

Comment:
I'm just wondering here, longer term, should we allow this to be set to match whatever branch you are testing? Or at least check if a branch with the same name exists on this repo, and if it does, use that?

If we are versioning the test repo along with the installer (master -> master, v1.3 -> v1.3) it seems likely that we would want to have this branch change depending on what is actually being tested. It would also allow us to update tests before they hit master in Delorean as we could create for example a 3scale-ga branch on the test repo that can be updated according to changes for the next ga release of 3scale.

Reply:
OK it could be possible, So what do you suggest? For example having a dropdown list with available branch names as an input param or something more sophisticated like trying to match the branch name? I would go with some simpler solution first.

Reply:
Personally if it was me, i'd change the test jobs to check the branch being specified existed and if it doesn't fall back to master, we used to do this for some rhmap stuff see https://github.com/feedhenry/fh-pipeline-library/blob/master/vars/checkoutGitRepo.groovy#L52-L61. The input to all these jobs for BRANCH would then be testOptions.gitBranch, but internally if that didn't exist it would use master.

I don't think doing the drop down initially makes much sense really, as most of this will be automated and we won't know if the branch will exist or not to specify it as an input.

I'm happy enough to leave it as "master" initially as long as we have a follow up jira to make it configurable.

Reply:
OK thanks for the tips. I think I will create a followup JIRA for this.

Reply:
Created https://issues.jboss.org/browse/INTLY-2098